scipy/optimize/_linprog_util.py

```diff
@@ -1080,25 +1036,21 @@ def vstack(blocks):
         zeros = np.zeros
         eye = np.eye
 
-    fixed_x = set()
```

Comment:
Can you explain what this was doing and how to get rid of it?

Reply:
`fixed_x` is a set of indices in the original `x`-array of variables which were fixed. This set is needed because `bounds` is still original here, but `_get_Abc()` needs the bounds of the unfixed variables only. `fixed_x` is used to remove the unnecessary values from `bounds`.
The removal is now done in a single statement `bounds = np.delete(bounds, undo[0], 0)`.

Reply:
OK, so this is just a more efficient way of doing it now that it is an array?